Userland/Applications/CalendarSettings/CalendarSettingsWidget.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2022-2022, Olivier De Cannière <olivier.decanniere96@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGUI/ComboBox.h>
+#include <LibGUI/SettingsWindow.h>
+
+class CalendarSettingsWidget final : public GUI::SettingsWindow::Tab {
+    C_OBJECT(CalendarSettingsWidget)
+
+public:
+    virtual void apply_settings() override;
+    virtual void reset_default_values() override;
+
+private:
+    CalendarSettingsWidget();
+    Vector<String> const m_weekday_names = { "Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday" };
+
+    String m_first_day_of_week { "Sunday"sv };
```

Comment:
Gotcha, looks like the way things played out, you're setting the value `m_first_day_of_week` to `m_first_day_of_week_combobox->text();` - the call to `text()` constructs a new `String`, which will go out of scope.

That's fine, but it seems like we don't need to store these strings at the class level, since they aren't used anywhere else? i.e. `apply_settings` could just be:
```c++
void CalendarSettingsWidget::apply_settings()
{
    auto first_day_of_week = m_first_day_of_week_combobox->text();
    auto default_view = m_default_view_combobox->text();
    Config::write_string("Calendar"sv, "View"sv, "FirstDayOfWeek"sv, first_day_of_week);
    Config::write_string("Calendar"sv, "View"sv, "DefaultView"sv, default_view);
}
```

And then `m_first_day_of_week` and `m_default_view` can go away.
Not a huge deal though.

Reply:
Yeah, I may have copied code without thinking enough... I'll fix it right away.